### ISSUE TITLE
Readme description of Installing pip2 on Ubuntu 20.04 does not work.

### ISSUE THREAD

**Issue by gatoWololo** [servo/servo#28143]
### Problem
Hello! The following part of the README.md is out of date:
```
If you are on Ubuntu 20.04 or higher you need to do the following to install the missing python2 bits:

curl https://bootstrap.pypa.io/get-pip.py -sSf -o get-pip.py
python2 get-pip.py
python2 -m pip install virtualenv
sudo apt install python-dev
```
This will result in the following error on Ubuntu 20.04:
```shell
> python2 get-pip.py
Traceback (most recent call last):
  File "get-pip.py", line 24244, in <module>
    main()
  File "get-pip.py", line 199, in main
    bootstrap(tmpdir=tmpdir)
  File "get-pip.py", line 82, in bootstrap
    from pip._internal.cli.main import main as pip_entry_point
  File "/tmp/tmpLFGhfw/pip.zip/pip/_internal/cli/main.py", line 60
    sys.stderr.write(f"ERROR: {exc}")
```
More information on why can be found [here](https://stackoverflow.com/questions/65869296/installing-pip-is-not-working-in-python-3-6)
### Solution
The following commands instead worked for me:
```shell
> curl -O https://bootstrap.pypa.io/2.7/get-pip.py
> python get-pip.py
> python -m pip install --upgrade "pip < 21.0"
> python2 -m pip install virtualenv
```

**Comment by MichaelMcDonnell** [comment]
I think this will be fixed by #28092 which switches to python 3.

**Comment by Apogeum12** [comment]
For Me working without issue, but I changed this step:
```bash
curl -O https://bootstrap.pypa.io/2.7/get-pip.py
> python get-pip.py
> python -m pip install --upgrade "pip < 21.0"
> python2 -m pip install virtualenv
```
on this:
```bash
sudo apt install python3-virtualenv python3-pip
```
And I installed some dependencies, i needed this:
```bash
sudo apt install gstreamer1.0-nice gstreamer1.0-plugins-bad \
		libunwind-dev libopencl-clang-dev librust-clang-sys+libloading-dev
```
And Bulid wihout error --dev version:
```bash
./mach bootstrap-gstreamer && ./mach build --dev
```
Personally, I would update README.md for ubuntu with the steps like me. Should be work this in general.
